src/js/playlist/track.ts

```diff
@@ -6,7 +15,7 @@
  * @property {boolean} default - Enable the track by default.
  */
 
-const Track = function(config) {
+const Track = function(config?: PlaylistItemTrack): PlaylistItemTrack | undefined {
```

Comment:
So literally anything can be passed in for `config`. I really is `any` or generic | undefined.

Should we use `PlaylistItemTrack | void` for the return value?

Reply:
hmm yeah there's some danger here. with the object assign, the types yell about passing in `GenericObject` when we are using object.assign on it (extra props). i guess I could use an interface here for `PlaylistItemTrack` but that also feels incorrect

Reply:
`PlaylistItemTrack` makes sense as an interface that extends `GenericObject`.